variants/pundit/spec/rails_helper.rb

```diff
@@ -0,0 +1,5 @@
+insert_into_file "spec/rails_helper.rb", after: /require "axe\/rspec"\n/ do
+  <<-REQUIRE
```

Comment:
:elephant: `<<~` instead of `<<-` will generate less leading whitespace so you can make your code line up with the code already in the file

Reply:
Thank you, I've changed `<<-` to `<<~`